spec/asciidoctor/validate_spec.rb

```diff
@@ -1,41 +1,41 @@
 require "spec_helper"
-  
+
 RSpec.describe Asciidoctor::BIPM do
   context "when xref_error.adoc compilation" do
-    around do |example|
-      FileUtils.rm_f "spec/assets/xref_error.err"
-      example.run
-      Dir["spec/assets/xref_error*"].each do |file|
-        next if file.match?(/adoc$/)
+    it "generates error file" do
+      File.write("xref_error.adoc", <<~"CONTENT")
+        = X
+        A
 
-        FileUtils.rm_f(file)
-      end
-    end
+        == Clause
 
-    it "generates error file" do
+        <<a,b>>
+      CONTENT
       mock_pdf
       expect do
         Metanorma::Compile
           .new
-          .compile("spec/assets/xref_error.adoc", type: "bipm", :"agree-to-terms" => true)
-      end.to(change { File.exist?("spec/assets/xref_error.err") }
-              .from(false).to(true))
+          .compile("xref_error.adoc", type: "bipm", no_install_fonts: true)
+      end.to(
+        change { File.exist?("xref_error.err") }
+          .from(false)
+          .to(true)
+      )
     end
   end
 
   it "validates committees" do
-      mock_pdf
-    FileUtils.rm_f "test.err"
```

Comment:
https://github.com/metanorma/metanorma-bipm/pull/115/files#diff-89eebfcbc0f14b6d989517837ca1e94fce4e2ce9a03233641cd936f2b8d2ed94R27 - this will help run tests in isolated tmp directory

Reply:
All file operations should be done in a temp directory.